src/index.html

```diff
@@ -1,17 +1,68 @@
 <!DOCTYPE html>
 <html lang="es">
-
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
```

Comment:
¿Podemos hacer este ajuste?

```suggestion
  <title>Tarjeta de usuario</title>
```

Reply:
Enterada.

Reply:
Se completo con este commit [d08eed6](https://github.com/MaxSteelAT/tarjeta-usuario/pull/1/commits/d08eed6b55117db7501e37725973b63927e5009c)